aspnetcore/security/authentication/social/microsoft-logins.md

```diff
@@ -53,19 +55,19 @@ Link sensitive settings like Microsoft `ClientId` and `ClientSecret` to your app
 
 ## Configure Microsoft Account Authentication
 
-Add the Microsoft Account service to `Startup.ConfigureServices`:
+Add the Microsoft Account service to the `Startup.ConfigureServices` method in the *Startup.cs* file:
 
-[!code-csharp[](~/security/authentication/social/social-code/StartupMS.cs?name=snippet&highlight=10-14)]
+[!code-csharp[](~/security/authentication/social/social-code/3.x/StartupMS3x.cs?name=snippet&highlight=10-14)]
 
 [!INCLUDE [default settings configuration](includes/default-settings.md)]
 
 [!INCLUDE[](includes/chain-auth-providers.md)]
 
-See the [MicrosoftAccountOptions](/dotnet/api/microsoft.aspnetcore.builder.microsoftaccountoptions) API reference for more information on configuration options supported by Microsoft Account authentication. This can be used to request different information about the user.
+For more information about configuration options supported by Microsoft Account authentication, see the [MicrosoftAccountOptions](/dotnet/api/microsoft.aspnetcore.builder.microsoftaccountoptions) API reference. This can be used to request different information about the user.
```

Comment:
For more information about configuration options supported by Microsoft Account authentication, see the [MicrosoftAccountOptions](/dotnet/api/microsoft.aspnetcore.builder.microsoftaccountoptions) API reference. `MicrosoftAccountOptions` can be used to request different information about the user.

Reply:
Acyrolix should have caught that. Make sure you run Acyrolix.